src/react-components/ory/user-logout-card.tsx

```diff
@@ -0,0 +1,65 @@
+import { gridStyle } from "../../theme"
+import { Button } from "../button"
+import { Card } from "../card"
+import { Typography } from "../typography"
+
+import { OAuth2Client, OAuth2ConsentRequest } from "@ory/client"
+
+import { Checkbox } from "../checkbox"
+import { Divider } from "../divider"
+
+export type UserLogoutCardProps = {
+  csrfToken: string
+  challenge: string
+  action: string
+  className?: string
+  cardImage?: string | React.ReactElement
+}
+
+export const UserLogoutCard = ({
+  csrfToken,
+  challenge,
+  action,
+  className,
+  cardImage,
+}: UserLogoutCardProps) => {
+  return (
+    <Card
+      className={className}
+      heading={
+        <div style={{ textAlign: "center" }}>
+          <Typography>Do you wish to log out?</Typography>
+        </div>
+      }
+      image={cardImage}
+    >
+      <form action={action} method="post">
+        <input type="hidden" name="_csrf" value={csrfToken} />
+        <input type="hidden" name="challenge" value={challenge} />
+        <div className={gridStyle({ gap: 16 })}>
+          <div
+            className={gridStyle({ direction: "row" })}
+            style={{ justifyContent: "space-between", alignItems: "center" }}
+          >
+            <Button
+              type="submit"
+              id="reject"
+              value="No"
+              name="submit"
+              variant="error"
+              header="No"
+            />
+            <Button
+              type="submit"
+              id="accept"
+              value="Yes"
+              name="submit"
+              variant="semibold"
+              header="Yes"
+            />
+          </div>
+        </div>
+      </form>
```

Comment:
Would we always manually be passing down the `challenge` and `csrftoken` to this component? Couldn't we just use the flow data, or if there isn't any, at least just a common type from the sdk?

Maybe we could rather extend this helper component? 
https://github.com/ory/elements/blob/main/src/react-components/ory/helpers/user-auth-form.tsx

The `UserAuthForm` already adds csrf and other hidden fields based on the flow data.

Reply:
This card is not part of the Kratos flows, but implements the UI for the Hydra/OAuth 2 logout flow. The card will be initialized in the hosted UI here: https://github.com/ory-corp/cloud/pull/4176/files#diff-53a791f9f8393dc7f05d73059e3f2f4380ee68316b834ed244a57905d63c87f6R43-R49, but this is not part of any SDK, so we don't have a common type here. I just used the same parameters as the hydra-login-consent-node here: https://github.com/ory/hydra-login-consent-node/blob/master/views/logout.pug